.circleci/config.yml

```
@@ -116,13 +116,31 @@ commands:
 
 
 jobs:
+  check-release-sbf:
+    docker:
+      - image: public.ecr.aws/certora/cvt-image:2024.10.16-4480-c8870b3
+    resource_class: small
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run:
+          name: Install Python deps
+          command: |
```

for a single command we can use it in a single line
```suggestion
          command: pip3.11 install requests semver
```

---

done